pkg/errcode/errcode.go

```diff
@@ -26,12 +53,12 @@ func GetCode(e error) int {
 	return err.code
 }
 
-// Create new errCode
+// NewErr creates new errCode with provided text.
 func NewErr(code int, text string) error {
 	return &errCode{code: code, text: text}
 }
 
-// Create new errCode with formated text.
+// Errorf creates new errCode with formated text.
 func Errorf(code int, format string, a ...interface{}) error {
 	return &errCode{code: code, text: fmt.Sprintf(format, a...)}
```

Comment:
nit: This could have been implemented as : 
```
return NewErr(code, fmt.Sprintf(format, a...))
```

Reply:
or just remove NewErr altogether if it has no real use.
as its the same as calling `Errorf(code, text)`